src/main/modules/i18n/index.ts

```diff
@@ -8,12 +7,12 @@ export class LanguageToggle {
   private supportedLanguages = ['en', 'cy'];
 
   public enableFor(app: Express): void {
-    app.use((req: ReqWithSession, res: Response, next: NextFunction) => {
+    app.use((req, res, next) => {
```

Comment:
dont we define types in typescript, wouldnt it be best to have its type?

Reply:
I was getting an error here since the line above is `app: Express` which provides the types the `app.use()` method and by retyping the function call here, I was getting an error saying that the retyped arguments are incompatible the existing `app.use()` type.